src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageClientMiddleLayer.cs

```diff
@@ -17,70 +13,16 @@ namespace Microsoft.VisualStudio.LanguageServerClient.Razor
-        private readonly LSPEditorService _editorService;
-
-        [ImportingConstructor]
-        public DefaultRazorLanguageClientMiddleLayer(LSPDocumentManager documentManager, LSPEditorService editorService)
-        {
-            if (documentManager is null)
-            {
-                throw new ArgumentNullException(nameof(documentManager));
-            }
-
-            if (editorService is null)
-            {
-                throw new ArgumentNullException(nameof(editorService));
-            }
-
-            _documentManager = documentManager;
-            _editorService = editorService;
-        }
-
-        public override bool CanHandle(string methodName)
-        {
-            return methodName == Methods.TextDocumentOnTypeFormattingName;
-        }
+        public override bool CanHandle(string methodName) => false;
 
         public override Task HandleNotificationAsync(string methodName, JToken methodParam, Func<JToken, Task> sendNotification)
         {
-            return null;
+            return Task.CompletedTask;
         }
 
-        public override async Task<JToken> HandleRequestAsync(string methodName, JToken methodParam, Func<JToken, Task<JToken>> sendRequest)
+        public override Task<JToken> HandleRequestAsync(string methodName, JToken methodParam, Func<JToken, Task<JToken>> sendRequest)
         {
-            if (methodName == Methods.TextDocumentOnTypeFormattingName)
-            {
-                var emptyResult = JToken.FromObject(Array.Empty<TextEdit>());
-                var requestParams = methodParam.ToObject<DocumentOnTypeFormattingParams>();
-                if (requestParams.Options.OtherOptions == null)
-                {
-                    requestParams.Options.OtherOptions = new Dictionary<string, object>();
-                }
-
-                requestParams.Options.OtherOptions[LanguageServerConstants.ExpectsCursorPlaceholderKey] = true;
-                var token = JToken.FromObject(requestParams);
-                var result = await sendRequest(token).ConfigureAwait(false);
-                var edits = result?.ToObject<TextEdit[]>();
-                if (edits == null || edits.Length == 0)
-                {
-                    return emptyResult;
-                }
-
-                if (!_documentManager.TryGetDocument(requestParams.TextDocument.Uri, out var documentSnapshot))
-                {
-                    return emptyResult;
-                }
-
-                await _editorService.ApplyTextEditsAsync(requestParams.TextDocument.Uri, documentSnapshot.Snapshot, edits).ConfigureAwait(false);
-
-                // We would have already applied the edits and moved the cursor. Return empty.
-                return emptyResult;
-            }
-            else
-            {
-                return await sendRequest(methodParam).ConfigureAwait(false);
-            }
+            throw new NotImplementedException();
```

Comment:
?

Reply:
Technically we should never be called because we return false for "canHandle" every time